src/Servers/Kestrel/Core/src/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -86,52 +88,65 @@ private static unsafe ulong GetMaskAsLong(byte[] bytes)
         }
 
         // The same as GetAsciiStringNonNullCharacters but throws BadRequest
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I think it's OK to inline this methods, as in effect it's just the delegate invocation.